src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * 4;
+    }
+
+    @Override
+    public void moreInfo() {

```

Comment:
The method name 'moreInfo' does not conform to Java naming conventions and does not clearly describe the action the method performs. Consider renaming it to something more descriptive, such as 'printInfo' or 'displayInfo'.